node/src/actors/chain_manager/mod.rs

```diff
@@ -807,15 +808,19 @@ fn update_reputation(
 }
 
 fn show_info_tally(tally_tx: &TallyTransaction, block_epoch: Epoch) {
-    let result = RadonTypes::try_from(tally_tx.tally.as_slice())
-        .map(|x| x.to_string())
-        .unwrap_or_else(|_| "RADError".to_string());
+    let result = RadonTypes::try_from(tally_tx.tally.as_slice());
+    let result_str = RadonReport::from_result(result, &ReportContext::default())
+        .map(|x| match x.into_inner() {
+            Ok(rad_types) => rad_types.to_string(),
+            Err(e) => format!("RadError: {:?}", e),
+        })
+        .unwrap_or_else(|_| "Unexpected RADError".to_string());
```

Comment:
Maybe you can show the RADError here?

```suggestion
        .unwrap_or_else(|e| format!("Unexpected RADError: {}", e);
```